Dockerfile.php5.6-busybox

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu:xenial as builder
+
+ENV PHP_VERSION 5.6.31
+ENV PECL_MODULES imagick-3.4.3 redis-3.1.3 memcached-2.2.0 ssh2-0.13 msgpack-0.5.7 igbinary-2.0.1 runkit-1.0.4
+
+ADD scripts/*.sh /usr/local/bin/
+
+RUN set -ex; \
+	\
+ 	apt-get update && apt-get install -y --no-install-recommends \
+		autoconf \
+		build-essential \
+		ca-certificates \
+		curl \
+		dirmngr \
+		dpkg-dev \
+		file \
+		g++ \
+		gcc \
+		gettext \
+		gnupg2 \
+		libbsd-dev \
+		libbz2-dev \
+		libc-client2007e-dev \
+		libc-dev \
+		libcurl3 \
+		libcurl3-dev \
+		libedit-dev \
+		libedit2 \
+		libicu-dev \
+		libkrb5-dev \
+		libldap2-dev \
+		libmagick++-dev \
+		libmcrypt-dev \
+		libmemcached-dev \
+		libpcre3-dev \
+		libsqlite3-0 \
+		libssh2-1-dev \
+		libtinfo-dev \
+		libtool \
+		libxml2 \
+		libxslt1-dev \
+		make \
+		mlocate \
+		pax-utils \
+		pkg-config \
+		re2c \
+		wget \
+		xz-utils \
+ 	; \
+	# Fix for missing library, required later during php configure/make.
+	cp /usr/lib/libc-client.so.2007e.0 /usr/lib/x86_64-linux-gnu/libc-client.a;
+
+ENV PHP_INI_DIR /usr/local/etc/php
+RUN mkdir -p $PHP_INI_DIR/conf.d
+
+ENV PHP_EXT_PATH=/usr/local/lib/php/extensions/no-debug-non-zts-20131226
+ENV PHP_CFLAGS="-fstack-protector-strong -fpic -fpie -O2"
+ENV PHP_CPPFLAGS="$PHP_CFLAGS"
+ENV PHP_LDFLAGS="-Wl,-O1 -Wl,--hash-style=both -pie"
+ENV GPG_KEYS 0BD78B5F97500D450838F95DFE857D9A90D90EC1 6E4F6AB321FDC07F2C332E3AC2BF0BC433CFC8B3
+
+ENV PHP_URL="https://secure.php.net/get/php-$PHP_VERSION.tar.xz/from/this/mirror" PHP_ASC_URL="https://secure.php.net/get/php-$PHP_VERSION.tar.xz.asc/from/this/mirror"
+ENV PHP_SHA256="c464af61240a9b7729fabe0314cdbdd5a000a4f0c9bd201f89f8628732fe4ae4" PHP_MD5=""
```

Comment:
Not a requirement, but this should be up with the version I think. Otherwise when you change the version you may miss that this needs updated until it fails the first time. Also this could be an arg so you don't have to change the file.